### PR TITLE
Deleted hidden input causing extraneous query. closes #398

### LINKED ISSUE #398: Selecting a different artifact version leads to a Not-Found page

### PR DESCRIPTION
**Pull request by travis032654**
@julienrf I tested it manually and it seems to work.

### CODE REVIEW

File: template/src/main/twirl/ch.epfl.scala.index.views/project/headproject.scala.html [hunk 1]
```diff
@@ -87,16 +87,21 @@
               name="artifact"
               onchange="this.form.submit()"
               data-live-search="true"
-              class="selectpicker artifact-name" data-style="btn-primary">
+              class="selectpicker artifact-name" data-style="btn-pr imary">
```
Comment (julienrf) [inline template/src/main/twirl/ch.epfl.scala.index.views/project/headproject.scala.html]: typo?

File: template/src/main/twirl/ch.epfl.scala.index.views/project/headproject.scala.html [hunk 2]
```diff
@@ -87,16 +87,21 @@
...
             }
+
             </select>
           </form>
           
           <form action="/@project.reference.organization/@project.reference.repository" action="GET">
+            @* @previouslySelectedTarget match {
+              case Some(target) =>  {<input type="hidden" name="target" value="@target">}
+              case None => {}
+            }*@
```
Comment (julienrf) [inline template/src/main/twirl/ch.epfl.scala.index.views/project/headproject.scala.html]: Looks good, why did you comment it?

**Comment by julienrf** [comment]
I think this is the right approach :)

**Comment by travis032654** [comment]
Oops! I don't have syntax highlighting so I missed that it was commented. Fixed it.

File: template/src/main/twirl/ch.epfl.scala.index.views/project/headproject.scala.html (hunk 1 above)
Comment (julienrf) [inline template/src/main/twirl/ch.epfl.scala.index.views/project/headproject.scala.html]: You still didn’t address this comment…

**Pull request merged**